onnxruntime/core/providers/cpu/ml/svmclassifier.cc

```diff
@@ -99,24 +117,34 @@ Status SVMClassifier<T>::Compute(OpKernelContext* ctx) const {
   {
     int64_t current_weight_0 = n * stride;
     int64_t maxclass = -1;
-    double maxweight = 0.f;
     std::vector<float> decisions;
     std::vector<float> scores;
     std::vector<float> kernels;
```

Comment:
>kernels [](start = 23, length = 7)

Because `vector_count_` is known, we can (pre-)allocate the whole vector here.

Reply:
see next answer.